lbz/misc.py

```diff
@@ -100,5 +115,8 @@ def wrapped(*args, **kwargs):
     return wrapped
 
 
-def copy_without_keys(data: dict, *keys) -> dict:
+def copy_without_keys(data: Union[dict, MultiDict], *keys) -> dict:
```

Comment:
I am not 100% sure if this method should accept `MultiDict` - it will be transformed to a normal dict and lose some features because of that 🤔

Reply:
we are already using it in that way.